src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -280,5 +334,66 @@ public TextEditor TextEditor
                 }
             }
         }
+
+        /// <summary>
+        /// The AnchorChanged event arguments.
+        /// </summary>
+        /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class AnchorTouchedEventArgs : EventArgs
```

Comment:
Please check warning message.

Reply:
AnchorTouchedEventArgs is commonly used in TextLabel, TextField, and TextEditor.
So I placed this in a new file TextEvent.cs.
I think, later we can implement common EventArgs for Text components like TextChanged here.